index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Audiowide&family=Exo+2:ital,wght@1,300;1,500;1,600&family=Open+Sans:wght@500&display=swap" rel="stylesheet">
+  <link href="stylesheets/style.css" rel="stylesheet"/>
+  <title>Welcome Page</title>
+</head>
+
+<body>
+  <nav>
+    <ul class="nav">
+      <li class="link-1"><a class= "nav-link" href="about.html">About Me</a> </li>
+      <li class="link-2"><a class= "nav-link" href="portfolio.html">Projects</a></li>
+    </ul>
+  </nav>
+      <header class="header">
+        <h2>Welcome Page
+        </h2>
+      </header>
+    <div class="main-page-content">
+    <aside class="left-column mountain"> 
+      <img src="images/stephen-ellis-NHvT7LMWI4A-unsplash.jpg" alt="Blue colored mountains">
+    </aside>
+    <main>
+      <div class="my-info">
+      <p id="text">
+        Hi! My name is Marlyn Lopez. 
+        I am an aspiring software developer residing in the West Coast with a background in audio production. 
+        Please click <a id="link" href="about.html">here </a> if you're curious to know more! 
+      </p>
+      </div>
```

Comment:
Style: This `div` is unnecessary since there's nothing else inside of `main`.

In general (if you have time) it's good to do a cleanup pass to see if you can eliminate any elements that aren't meaingful like `div`s.